changelogs/fragments/371-add-bridge-port-cost-mode.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - api - Add port-cost-mode to /interface/bridge which is supported since RouterOS 7.13
```

Comment:
```suggestion
  - api_info, api_modify - add ``port-cost-mode`` to ``interface bridge`` which is supported since RouterOS 7.13 (https://github.com/ansible-collections/community.routeros/pull/371).
```